include/GTID_Server_Data.h

```diff
@@ -31,6 +31,8 @@ class GTID_Server_Data {
 	void dump();
 };
 
+bool addGtid(const std::string& uuid, const gtid_interval_t &iv, gtid_set_t& gtid_executed);
+bool addGtid(const gtid_t& gtid, gtid_set_t& gtid_executed);
 bool addGtidInterval(gtid_set_t& gtid_executed, std::string server_uuid, int64_t txid_start, int64_t txid_end);
```

Comment:
Both of these old functions have only one reference in the codebase. It is better to update those call sites to use the new function rather than maintaining multiple aliases.